README.md

```diff
@@ -324,33 +324,32 @@ Explicit:
 Unchecked
 ```
 
-Once your Google account is registered and the music is uploaded, extract the cookies for our test suite via:
+Once your Google account is registered and the music is uploaded, set the following environment variables
 
 ```
-# Enter into the node REPL
-node
-
-# Inside of the REPL, dump our cookies
-var browser = require('wd').remote();
-browser.init({browserName: 'chrome'}, console.log);
-// Wait for browser window to open
-browser.get('https://play.google.com/music/listen', console.log);
-// Wait for redirect to accounts.google.com
-// Manually log in to page
-// When you are logged in to Google Music, dump the output of the following into `test/cookies.json`
-browser.allCookies(function (err, cookies) { fs.writeFileSync('test/cookies.json', JSON.stringify(cookies, null, 4), 'utf8'); });
+GPM_USER = {your_google_account_username}
+GPM_PWD = {your_google_account_password}
 ```
 
+This is so the test suite can automate the login process to test against Google Play Music
+
 Finally, we can run the test suite:
 
 ```bash
 # Start up a Selenium server
-npm run webdriver-manager-start &
+npm install -g selenium-standalone
+selenium-standalone install
+selenium-standalone start
```

Comment:
As a side note, there's now https://github.com/pose/webdriver-manager as well which is an extracted version of what was in `protractor`